.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/antonbabenko/pre-commit-terraform
+    rev: v1.77.0
+    hooks:
+      - id: terraform_fmt
+      - id: terraform_validate
+      - id: terraform_docs
+# TODO: Fix all checkov errors before enabling
```

Comment:
Entirely remove checkov, or embed it, lets not add commented code to the main branch.